fw/cache.c

```diff
@@ -3222,11 +3220,12 @@ static int
 tfw_cache_start(void)
 {
 	int i, r = 1;
-	TfwGlobal *g_vhost = tfw_vhost_get_global();
+
+	BUG_ON(cache_cfg.cache == TFW_CACHE_UNDEFINED);
```

Comment:
```suggestion
	if (WARN_ON_ONCE(cache_cfg.cache == TFW_CACHE_UNDEFINED))
	    return some_error_code;
```

Let's not crash server without a good reason!